test/node/object_id.test.ts

```diff
@@ -336,22 +331,9 @@ describe('ObjectId', function () {
     done();
   });
 
-  it('should throw if a 12-char length but non-12 byte string is passed in', function () {
-    const characterCodesLargerThan256 = 'abcdefŽhijkl';
-    const length12Not12Bytes = '🐶🐶🐶🐶🐶🐶';
-    expect(() => new ObjectId(characterCodesLargerThan256).toHexString()).to.throw(
-      BSONError,
-      'Argument passed in must be a string of 12 bytes'
-    );
-    expect(() => new ObjectId(length12Not12Bytes).id).to.throw(
-      BSONError,
-      'Argument passed in must be a string of 12 bytes'
-    );
-  });
-
-  it('should have isValid be true for 12-char length and 12-byte length string', function () {
+  it('should have isValid be false for 12-char length and 12-byte length string', function () {
```

Comment:
We should break out the `isValid` test strings with 2-byte unicode code points separate from this test

Reply:
I organized the isValid tests per input type and covered relevant cases